shrine/README.md

```diff
@@ -0,0 +1,45 @@
+# Shrine
+
+A sample shopping app that uses Material widgets in its UI and
+[`scoped_model`](https://pub.dartlang.org/packages/scoped_model) to
+manage the state of its shopping cart.
+
+## Goals
+
+* Show how to customize Flutter's Material Compoenent widgets to produce
```

Comment:
Spelling of `Component`

Reply:
Nice catch. I think that's the German spelling. :)

Fixed.